Dockerfile

```diff
@@ -24,7 +24,7 @@ COPY ./app /opt/$NAME/app
 RUN chown $USER:$USER /opt/$NAME
 
 # Tell Docker we are going to use this ports
-EXPOSE 3050
+EXPOSE 30567
```

Comment:
this will cause problems. there are better ways to map ports throught docker-compose

Reply:
Good catch, reverting this and on the other MSs to avoid AWS issue.